### DISCUSSION TITLE
ioBroker.roborock stürzt ständig ab

### DISCUSSION THREAD

**Discussion by jahnbes** [copystring/ioBroker.roborock#162] [Q&A]
Hallo, 
meine Installation von copystring/ioBroker.roborock stürzt nach wenigen Sekunden ständig ab. Das Protokoll eines solchen Absturzes findet Ihr hier:
[roborock_iobroker_error_20230603_1438.docx](https://github.com/copystring/ioBroker.roborock/files/11645383/roborock_iobroker_error_20230603_1438.docx)
und als pdf:
[roborock_iobroker_error_20230603_1438.pdf](https://github.com/copystring/ioBroker.roborock/files/11645403/roborock_iobroker_error_20230603_1438.pdf)

Ich bin daraus nicht schlau geworden, vielleicht kann sich jemand dessen erbarmen und mal einen Blick drauf werfen und mir einen Tipp geben? Braucht Ihr weitere Angaben von mir dazu?
Die Karte habe ich kurzfristig auch schon im Browser ´gesehen, auch die Fahrspur, aber dann knallt es leider immer und aus ist.

Iobroker nutze ich seit langem, den Roborock S7 MAXV habe ich erst seit einem halben Jahr, den entsprechenden Adapter habe ich vor einem Monat installiert, seitdem suche ich den Fehler. Der ioborker läuft bei mir auf einem raspberry Pi 4 mit 8GByte HS, also eigentlich ausreichend. Der zeigt auch keinen Fehler an.

Im Voraus vielen Dank und Grüße, Bert.

**Comment by copystring** [comment]
Hi,

ist die Kartenerstellung auf dem Pi aktiviert?

**Reply by jahnbes** [reply]
Ja, die Kartendarstellung ist auf dem Pi aktiviert, genauer gesagt im iobroker, der auf dem Pi läuft.

**Reply by copystring** [reply]
Diese Funktion läuft auf einem Pi nicht. Du musst das deaktivieren.

**Reply by jahnbes** [reply]
Habe ich getan, mal sehen, was passiert, bis jetzt jedenfalls läuft de Adapter.
